chef-config/lib/chef-config/fips.rb

```diff
@@ -1,6 +1,6 @@
 #
 # Author:: Matt Wrock (<matt@mattwrock.com>)
-# Copyright:: Copyright (c) 2016 Chef Software, Inc.
+# Copyright:: Copyright (c) 2016-2018, Chef Software Inc.
```

Comment:
I think your copyright bumping script isn't sure what year it is

Reply:
ugh, that must be my laptop

Reply:
actually that's accurate, i worked on this last year and rebased it....